AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -371,6 +370,23 @@ private IContainer CreateIoCContainer(IServiceCollection services)
                 await sampleData.CreateAdminUser();
             }
         }
+
+        private static void CheckForHangfireDatabaseAndCreateIfItDoesNotExist()
+        {
+            using (var sqlConnection = new SqlConnection("Server=(localdb)\\MSSQLLocalDB;Integrated Security=true;MultipleActiveResultsets=true;"))

```

Comment:
This should be using the connection string from config. Not every developer will be using the same connection string